pyproject.toml

```diff
@@ -54,7 +54,8 @@ imaging = [
     'gcsfs',
     'xarray-multiscale',
     'parameterized',
-    'zarr==2.13.3',
+    'zarr',
+    'ome-zarr',
```

Comment:
Should we pin the ome-zarr version here? Just in case they add a new feature that breaks the writer. Also, same comment for zarr.

Reply:
versions pinned 👍